app/assets/javascripts/discourse/app/controllers/application.js

```diff
@@ -31,6 +30,11 @@ export default Controller.extend({
     );
   },
 
+  @discourseComputed
+  sidebarAvailable() {
```

Comment:
```suggestion
  canDisplaySidebar() {
```

Perhaps the above instead? available is alittle vague IMO.